### PR TITLE
smaller header

### PR DESCRIPTION
**Pull request by boygirl**
@paulathevalley here's another option that just drastically cuts down the height of the header on every page: 
screen shot 2018-06-24 at 9 47 39 pm

Does this seem better to you?

### CODE REVIEW

**Comment by paulathevalley** [comment]
This header height looks good to me. I’m starting to wonder whether the Victory header needs to keep the Formidable-space branding, because the darkness of both make them seem a little _too_ related. 

screen shot 2018-06-25 at 9 32 53 am

Colors aren’t perfect, but what do you think? I think this could also support the layout in #554 if tightening it up even more is important. I noticed the `.Page` class will need its height attribute updated to `height: calc(100% - 190px);`.

I am also wondering whether the Victory header needs to come from `formidable-landers` anymore. The universal Formidable header has clear Formidable branding with it, and these headers seem more useful for open source pages that don’t have their own branding yet. It’d probably be easier for you to code up, too. This is probably a @beccalee123 or @coopy question, though.

**Comment by boygirl** [comment]
@paulathevalley thank you! I'll try both with a lighter background.

**Comment by boygirl** [comment]
@paulathevalley I ended up adding a 1px bottom border to the header because it was looking very odd when scrolling through code with nearly the same color background. Here's what it looks like:
screen shot 2018-06-25 at 4 49 54 pm
screen shot 2018-06-25 at 4 49 34 pm

**Comment by paulathevalley** [comment]
Nice! :rocket:

File: src/styles/components/footer.css [hunk 1]
```diff
@@ -1,9 +1,12 @@
 .formidableFooter.victory {
   margin: 0;
   padding: 0;
+  background-color: var(--lightestSand);
 }
 
 .formidableFooter.victory .formidableFooter-container {
   margin-right: 3vw;
   margin-left: 3vw;
+  margin-bottom: 10;
```
Comment (paulathevalley) [inline src/styles/components/footer.css]: 10px?

Comment (boygirl) [reply src/styles/components/footer.css]: oops, that should be taken out!

**Pull request merged**